tasks/create_queue.py

```diff
@@ -0,0 +1,73 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from __future__ import print_function
+
+import argparse
+
+
+def create_queue(project,
+                 queue_name,
+                 location):
+    # [START cloud_tasks_create_queue]
+    """Create a task queue."""
```

Comment:
Please document the args: https://github.com/GoogleCloudPlatform/python-docs-samples/blob/master/AUTHORING_GUIDE.md#documenting-arguments

Reply:
@averikitsch The doc mentions that it's rarely necessary to exhaustively document the args. Should I just make the variable names more descriptive or is this a case where I would need to document the args in the docstring?

Reply:
Actually, just move the region tag above the method signature so users know what the args are passed into the function